### DISCUSSION TITLE
Error when using OpenSearch and LocalAI

### DISCUSSION THREAD

**Discussion by genwch** [FlowiseAI/Flowise#597] [General]
I have 2 similar 'Chatflows'
fig1 - use in-memory vector store
fig2 - use open search vector store
but get difference result. 
when using in-memory, it is same as expected.
when using open search, it returned 'Illegal invocation'
attached sample text file for easy debug.

[fig1]
<img width="1050" alt="Screenshot 2023-07-22 at 5 31 57 PM" src="https://github.com/FlowiseAI/Flowise/assets/18664789/92cbcce8-0086-4f26-9192-c8313610e048">
[fig2]
<img width="1063" alt="Screenshot 2023-07-22 at 5 32 51 PM" src="https://github.com/FlowiseAI/Flowise/assets/18664789/32ab9ed6-8b1a-4589-9500-170a2c46917b">
[sample.txt](https://github.com/FlowiseAI/Flowise/files/12136087/sample.txt)

**Comment by chungyau97** [comment]
Could you try OpenAI models with OpenSearch vector store?
